packages/editor/src/components/widgets/code-editor/useCodeEditor.ts

```diff
@@ -39,6 +39,38 @@ export const useMonacoEditor = (options?: { modifyAction?: ModifyAction; scriptA
     } else {
       const text = value.length > 0 && options?.modifyAction ? options.modifyAction(value) : value;
       editor.executeEdits('browser', [{ range: selection, text, forceMoveMarkers: true }]);
+      if (type === 'func') {
```

Comment:
this will no longer work if the other pr is merged right?

Reply:
Yes, that's right. i'll merge the function-browser first and then adapt the other pr accordingly. i didn't realize that i needed the type after all.

Reply:
not necessarily, you could aso invent another paremeter in this browser value type, e.g. like selectFunctionParam